crates/containerd-shim-wasmedge/src/main.rs

```diff
@@ -2,5 +2,5 @@ use containerd_shim_wasm::sandbox::cli::{revision, shim_main, version};
 use containerd_shim_wasmedge::WasmEdgeInstance;
 
 fn main() {
-    shim_main::<WasmEdgeInstance>("wasmedge", version!(), revision!(), None);
+    shim_main::<WasmEdgeInstance>("wasmedge", version!(), revision!(), Some("v1"), None);
```

Comment:
```suggestion
    shim_main::<WasmEdgeInstance>("wasmedge", version!(), revision!(), "v1", None);
```

Same on the other shims.